code/client/munkilib/FileRepo.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/python
+# encoding: utf-8
+#
+# Copyright 2016 Centrify Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+FileRepo
+Created by Centrify Corporation 2016-06-02.
+Implementation for accessing a repo via direct file access, including
+a remote repo mounted via AFP, SMB, or NFS.
+"""
+
+from collections import namedtuple
+import sys
+sys.path.append("/usr/local/munki/munkilib")
```

Comment:
None of the other Munki Python code is tied to a specific install location -- it all works with relative imports. I'd like that to continue to be the case. This also helps when testing -- I can leave the "production" versions of the Munki tools installed in /usr/local/munki, but test new code anywhere in working space.